### DISCUSSION TITLE
vscode extension for auto import ?

### DISCUSSION THREAD

**Discussion by magic-thomas** [adonisjs/core#3881] [Help] [answered]
I am looking for some vscode extension for auto import with shortcut .

How can I import Model / Controller  with shortcut  in vscode ?

**Comment by Julien-R44** [comment]
Automatic import is already possible with Typescript and Vscode
What do you mean by "shortcut"? Can you explain a bit more precisely what you want? I'm not sure I understand.

**Reply by magic-thomas** [reply]
I want this to be inserted by typing keyboard shortcut.

```js
import User from 'App/Models/User'
```

I tried 'Ctrl + Option + i' for this. 

But it shows error even though I have  'app\Models\User.ts' . 

```sh
The class is not found.
```



I am from Laravel and that keyboard shortcut is from one of Laravel extensions, I guess.

Anyway I want to insert the 'import' statement with keyboard shortcut, and wish I explained  correctly what I think . 

Thanks.

**Comment by magic-thomas** [comment, marked as answer]
I found solution. 

I installed 'typescript importer' 

[https://marketplace.visualstudio.com/items?itemName=pmneo.tsimporter](url)


and I set shortcut keys. It works good. 

Thanks.
